src/keylessBackup/KeylessBackupPhoneCodeInput.test.tsx

```diff
@@ -29,21 +46,88 @@ describe('KeylessBackupPhoneCodeInput', () => {
     expect(getByText('phoneVerificationInput.help')).toBeTruthy()
     expect(getByTestId('PhoneVerificationCode')).toBeTruthy()
     expect(getByTestId('KeylessBackupPhoneCodeInputClose')).toBeTruthy()
+    await waitFor(() => expect(mockFetch).toHaveBeenCalledTimes(1))
+    expect(mockFetch).toHaveBeenCalledWith(`${networkConfig.cabIssueSmsCodeUrl}`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: '{"phoneNumber":"+15555555555","clientPlatform":"android","clientBundleId":"org.celo.mobile.debug"}',
+    })
+  })
+
+  it('displays an error if verification code request fails', async () => {
```

Comment:
Should we have a test for when requesting is successful, but verification fails?

Reply:
`shows error in verifying sms code` test I believe covers this